tests/test_raster/test_raster.py

```diff
@@ -1057,17 +1049,13 @@ def test_is_modified(self, example: str) -> None:
         r = gu.Raster(example)
         assert not r.is_modified
 
-        # This should not trigger the hash
```

Comment:
Why are these changes necessary?

Reply:
Good point (I made some of these modifs back in September...). This was failing because casting with a Python scalar now changed the output `dtype` of the array.
I added the test back but replaced `0` by `np.array([0], dtype=r.dtype)` and that now works again.